### ISSUE TITLE
Bug 54110167: [React Native Gallery -> DatePicker]: While navigating in the calendar date picker, screen reader fails to announce current day information to user.

### ISSUE THREAD

**Issue by Yajur-Grover** [microsoft/react-native-gallery#499]
Build Details:
OS Version: Dev (OS Build 27695.1000)
App: React Native Gallery 
App Version: 1.0.18.0
React Native Windows Version: 0.75.0
AT: Screen Reader
 
Pre-Requisite:
Turn on Narrator (Ctrl + Win + Enter).
 
Repro-Steps:
Launch the React Native Gallery app.
Navigate to ‘All samples’ button present in left navigation pane and invoke it.
Navigate to ‘DatePicker’ button present under “Date & Time” group and invoke it.
Navigate to 'calendar date picker' button present under "A simple DatePicker" heading.
Navigate to current date i.e., today and listen to narrator announcement.
Actual Result:
While navigating in the calendar date picker, screen reader fails to announce current day information to user. When user navigates in it narrator is simply announcing as "25, non-selected, Column header Wednesday, Column 4 of 7".

Expected Result:
While navigating in calendar date picker, screen reader should announce the current day information to user. When user navigates to it narrator should announce as "25, Today, non-selected, Column header Wednesday, Column 4 of 7".

User Impact:
Users with visual impairment who rely on screen reader will not able to know about the today/current day information.

Link to ADO bug: https://dev.azure.com/microsoft/OS/_workitems/edit/54110167

**Comment by chrisglein** [comment]
This is literally hosting the WinUI2 date picker control. May be some differences in the accessibility tree based on how we host it.

**Comment by Yajur-Grover** [comment]
Looking at the WinUI 2 Gallery example CalendarView, the announcement from Narrator when focused on a specific date is the same as what is on React Native Gallery. In WinUI 3 Gallery, the announcement changes to include current day information.

**Comment by Yajur-Grover** [comment]
Closing as this has been marked resolved on ADO - logged on the WinUI team.
